Sources/Tracing/TracerProtocol.swift

```diff
@@ -59,6 +59,14 @@ public protocol Tracer: LegacyTracer {
         file fileID: String,
         line: UInt
     ) -> Self.Span
+
+    /// Retrieve the recording span for the given `ServiceContext`.
+    ///
+    /// - Note: This API does not enable look up of already finished spans.
+    /// It was added retroactively with a default implementation returning `nil` and therefore isn't guaranteed to be implemented by all `Tracer`s.
+    /// - Parameter context: The context containing information that uniquely identifies the span being obtained.
+    /// - Returns: The span identified by the given `ServiceContext` in case it's still recording.
+    func recordingSpan(identifiedBy context: ServiceContext) -> Span?
```

Comment:
I think this is okey, but I'd perhaps rename it. Names used by the linked libraries sound ok to me, especially `activeSpan(identifiedBy: context)` seems pretty good. 

I'm not sure about the 'record**ing**` because it makes it sound like not a getter somehow

Reply:
> I'm not sure about the 'recording` because it makes it sound like not a getter somehow

Yeah, good point. I haven't read it this way before, but I can see how it might be misunderstood. What about using `activeSpan(identifiedBy:)` here instead?

Reply:
I renamed it to `activeSpan(identifiedBy:)` now, LGTM.